server/internal/infrastructure/memory/model.go

```diff
@@ -179,3 +179,22 @@ func MockModelNow(r repo.Model, t time.Time) func() {
 func SetModelError(r repo.Model, err error) {
 	r.(*Model).err = err
 }
+
+func (r *Model) SaveAll(ctx context.Context, models model.List) error {
+	if r.err != nil {
+		return r.err
+	}
+	if len(models) == 0 {
+		return nil
+	}
+	if !r.f.CanWrite(models[0].Project()) {
+		return repo.ErrOperationDenied
+	}
```

Comment:
what if not all models on the same project?

Reply:
I was thinking of the same thing, I'm gonna check for that and return an error